registry.json

```diff
@@ -3656,5 +3656,19 @@
     "telegram": "hungbv2",
     "operatorName": "Bui Viet Hung",
     "github": "sirhung1993"
+  },
+  {
+    "ticker": "STKPL",
+    "poolId": "596371eefd3d7c3a201d12d64428b82a4c967bc7746d8a31350799ca",
+    "social": {
+      "website": "http://www.steelpoel.nl",
+      "twitter": "",
+      "facebook": "https://www.facebook.com/groups/602261507626219",
```

Comment:
Please could we see some social media activity? There are no members in the facebook group... Maybe adding twitter might be easier?